docs/app/insights.md

```diff
@@ -1,81 +1,199 @@
-4. Hover over the desired panel, click the Context Menu button ("..." icon), and click "Duplicate"
-5. Reposition and resize as needed
-6. Click the "Save" action button in the page header
+- **Visible** - Determines if the header is visible or not
+- **Name** - Sets a name in the header area
+- **Icon** - Sets a small icon by the name
+- **Color** - Sets a color for the icon
+- **Note** - Sets a short description of the icon
 
-## Deleting a Panel
+### Label Overview
 
-1. Click on the Insights module to go to the Dashboard listing
-2. Click the desired dashboard to view it
-3. Click the "Edit Panels" action button ("pencil" icon) in the header
-4. Hover over the desired panel, click the Context Menu button ("..." icon), and click "Delete Panel"
-5. Click the "Save" action button in the page header
+As the name implies, this panel simply allows you to _label_ a group of other panels. Labels do not provide analytics.
+Labels simply help visually group the other panels (_which do provide analytics_) and give context to your dashboard.
+
+#### Label Panel Options
+
+![Static image of a label options](../assets/insights-images/insights_panels_label-options.gif)
+
+- **Label** - Sets the label text
+- **Color** - Sets the label text color
+
+### List Overview
+
+List panels display some number of items from a collection, _as a list!_ These items displayed can be filtered by other
+criteria as well. So for example, you could show the site's top 5 selling products, 3 worst performing sales regions, 10
+students with the highest GPA _that are also_ in intramural sports, or create any other kind of ranked list that your
+dataset supports.
+
+#### List Panel Options
+
+![Image of List panel options](../assets/insights-images/insights_panels_list-panel-options.gif)
+
+- **Collection** - Selects the collection to rank
+- **Limit** - Limits the number of items that will appear in the dashboard
+- **Sort Field** - Sets the collection field you wish to order results by.
+- **Sort Direction** - Sets items to display in ascending or descending order
+- **Display Template** - Sets which fields will appear on-screen to represent list items
+- **Filter** - Sets filters to only consider and rank items that meet some criteria
+
+### Metrics Overview
+
+Metrics panels allow you to aggregate numeric fields into a single number.
+
+#### Metrics Panel Options
+
+![Image of Metrics panel options](../assets/insights-images/insights_panels_metrics-panel-options.gif)
+
+- **Collection** - Selects the collection
+- **Field** - Selects the field to run aggregate function on
+- **Aggregate Function** - Selects the type of aggregate to perform
+- **Sort Field** - Sets the collection field you wish to order results by
+- **Filter** - Sets filters to only only run metrics on items that meet some criteria
+
+::: warning Must be numeric Non-numeric fields like text cannot be aggregated, so they become grayed out and
```

Comment:
Again, let's update based on @jaycammarano's PR... I think certain functions just gray out based on the field.

Reply:
Yes.

Reply:
```
::: tip Why are some Fields grayed out?

Certain Fields cannot be used with certain aggregate functions. For example, you cannot use the average function on Fields that store text, because text cannot be averaged. When there is a conflict between panel options, Fields will be grayed out and un-selectable in the `field dropdown list`.

:::
```

Just confirming this is accurate.